kinto/plugins/accounts/authentication.py

```diff
@@ -1,10 +1,29 @@
 import bcrypt
 from pyramid import authentication as base_auth
 
+from kinto.core import utils
 from kinto.core.storage import exceptions as storage_exceptions
 
+from . import ACCOUNT_CACHE_KEY
+
 
 def account_check(username, password, request):
+    settings = request.registry.settings
+    hmac_secret = settings['userid_hmac_secret']
+    cache_key = utils.hmac_digest(hmac_secret, ACCOUNT_CACHE_KEY.format(username))
+    cache_ttl = settings.get('account_cache_ttl_seconds', 30)
```

Comment:
This setting should be documented and could be added to settings template IMO

Reply:
I guess you mean that https://github.com/Kinto/kinto/pull/1413/files#diff-2193261fa1533ff793bf90b447434bc8R32 is not sufficient?